src/Libraries/CoreNodeModels/HigherOrder/Apply.cs

```diff
@@ -73,6 +73,7 @@ public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode
     [NodeName("Function Compose")]
     [NodeCategory(BuiltinNodeCategories.CORE_EVALUATE)]
     [NodeDescription("FunctionComposeDescription", typeof(Resources))]
+    [OutPortTypes("Function")]
```

Comment:
@aparajit-pratap should this have a namespace or the classes defined . ds files don't have namespaces?